pkg/porter/create.go

```diff
@@ -1,43 +1,73 @@
 package porter
 
 import (
+	"errors"
 	"fmt"
 	"os"
 	"path/filepath"
+	"strings"
 
 	"get.porter.sh/porter/pkg"
 	"get.porter.sh/porter/pkg/config"
 )
 
-func (p *Porter) Create() error {
-	fmt.Fprintln(p.Out, "creating porter configuration in the current directory")
+// Create creates a new bundle configuration with the specified bundleName. A directory with the given bundleName will be created if it does not already exist.
+// If bundleName is the empty string, the configuration will be created in the current directory, and the name will be "porter-hello".
+func (p *Porter) Create(bundleName string) error {
+	// Normalize the bundleName by removing trailing slashes
+	bundleName = strings.TrimSuffix(bundleName, "/")
 
-	err := p.CopyTemplate(p.Templates.GetManifest, config.Name)
+	// If given a bundleName, create directory if it doesn't exist
```

Comment:
A couple of comments:
Instead of `if bundleName is not nil` and `ifBundle is nil` it's good to build the function around the concept of 
"If this thing, do this thing. Else, do the other thing"
You can use else, but the logic of the function should implicitly do the else.
Also, we should break this out into another function entirely. 

It would be preferred imo to not change the function signature of `Create()` to `Create(string)` but I have no strong preference.

Reply:
You could make this optional and have the function signature not change by using `…` for the param.